packages/venia-concept/src/components/Checkout/paymentsForm.css

```diff
@@ -3,28 +3,30 @@
 
 .body {
     composes: body from './form.css';
+    align-content: start;
+    background-color: white;
     grid-gap: 0.5rem;
-    height: calc(
-        var(--minicart-height) - var(--flow-footer-height) -
-            var(--minicart-footer-padding-bottom)
-    );
-    overflow-y: scroll;
+    height: 30rem;
```

Comment:
Are you making a conscious decision to not have the billing form span the height of the viewport? If so, I'm fine with this change, but we are definitely changing the way the payment form looks now.

[![Image from Gyazo](https://i.gyazo.com/783ab9be485c555f1e056cef407c1b57.gif)](https://gyazo.com/783ab9be485c555f1e056cef407c1b57)

Reply:
Also, why not just use `vh` units?

Reply:
The whole problem here is that mobile Safari doesn't understand that the "viewport" shouldn't include browser chrome. So `100vh` works fine in every other browser, but in Safari you end up with buttons sitting underneath the browser UI at the bottom of the screen.

Reply:
Ok -- can we maybe increase it a tiny bit? As is unchecking the box to add a distinct address doesn't cause the UI to change or move so it's a little bit of a weird UX. Maybe we should add an scroll/focus to the address form when the button is unchecked?

Reply:
Haha, I was just demoing this to @soumya-ashok and she had the same feedback. I've added an effect to scroll the new fields into view when added.